src/components/edit-button/index.jsx

```diff
@@ -3,10 +3,9 @@ import { useLanguage } from '../../lib/i18n';
 import style from './style.module.css';
 
 export default function EditThisPage({ isFallback }) {
-	const { url } = useLocation();
+	let { path } = useLocation();
 	const [lang] = useLanguage();
 
-	let path = url.replace(/\/$/, '') || '/index';
```

Comment:
The `/index` bit isn't needed as there's only two index pages on the site (`/` & `/tutorial`), neither of which show the edit button